src/App.css

```diff
@@ -151,3 +151,22 @@ li.panel .button-list {
 .side-pane__menu a {
   text-decoration: none;
 }
+
+@media (max-width: 400px) {
```

Comment:
can we use the template CSS instead ?

Reply:
Because the css template does not dela with phone size screens very well.
e.g. 
![image](https://user-images.githubusercontent.com/13916213/45481307-81a05a00-b74b-11e8-96cd-d53090547167.png)